### ISSUE TITLE
DiCE also supports model agnostic approaches to generate counterfactuals

### ISSUE THREAD

**Issue by gaugup** [christophM/interpretable-ml-book#261]
In section 6.1.5 (https://christophm.github.io/interpretable-ml-book/counterfactual.html#example-software), it is mentioned that the DiCE library only supports counterfactual explanations for differentiable models. 

We would need to update this documentation as DiCE also supports model agnostic methods for deriving counterfactual examples and explanations. More on this here:- https://github.com/interpretml/DiCE/blob/master/docs/source/notebooks/DiCE_getting_started.ipynb. 

Blurb from the notebook below:-
====
Model-Agnostic: These methods apply to any black-box classifier or regressor. They are based on sampling nearby points to an input point, while optimizing a loss function based on proximity (and optionally, sparsity, diversity and feasibility). Use this class of methods for sklearn models. Currently supported methods are:
Randomized Search
Genetic Search
KD Tree Search (for counterfactuals from a given training dataset) 
====

**Comment by christophM** [comment]
thanks, it's updated
